src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -45,80 +43,92 @@
             bottom-value (animation/create-value -250)
             opacity-value (animation/create-value 0)]
     {:component-did-mount #(send.animations/animate-sign-panel opacity-value bottom-value)}
-    [react/animated-view {:style (styles/animated-sign-panel bottom-value)}
-      [react/animated-view {:style (styles/sign-panel opacity-value)}
-       [react/view styles/signing-phrase-container
-        [react/text {:style styles/signing-phrase} signing-phrase]]
-       [react/text {:style styles/signing-phrase-description} (i18n/label :t/signing-phrase-description)]
-       [react/view styles/password-container
+    [react/animated-view {:style (send.styles/animated-sign-panel bottom-value)}
```

Comment:
i think we can use `styles` in views from this ns, only if we are using another ns styles we should use `something.styles`

Reply:
Why not.
If so we should document all of it as this is an exception to the rule we choosed.